src/parsers/problem/TheJobOverflowProblemParser.ts

```diff
@@ -0,0 +1,26 @@
+import { Sendable } from '../../models/Sendable';
+import { TaskBuilder } from '../../models/TaskBuilder';
+import { htmlToElement } from '../../utils/dom';
+import { Parser } from '../Parser';
+
+export class TheJobOverflowProblemParser extends Parser {
+  public getMatchPatterns(): string[] {
+    return ['http://thejoboverflow.com/problem/*'];
```

Comment:
TheJobOverflow also seems to support https, please include "https://thejoboverflow.com/problem/*" as well.